Documentation/project-docs/copyright.md

```diff
@@ -0,0 +1,34 @@
+Copyright
+=========
+
+The .NET Core copyright is held by ".NET Foundation and Contributors". 
+
+The [.NET Foundation](http://www.dotnetfoundation.org/) is an independent organization that encourages open development and collaboration around the .NET ecosystem.
```

Comment:
Got the text from here: https://dotnetfoundation.org/about

I assume it is intended to mean independent from Microsoft, which pays the majority of the bills for the foundation.